environment.yml

```diff
@@ -12,7 +12,12 @@ dependencies:
   - cpuonly
   - pytorch
   - torchvision
+  - pandas
+  - scipy
+  - numpy
+  - pynetdicom
```

Comment:
I don't think you actually use pynetdicom, but it doesn't really matter if we keep these new dependencies.